lib/redmine_email_inline_images/mail_handler_patch.rb

```diff
@@ -15,11 +15,11 @@ module InstanceMethods
       # an issue created by an email request
       def plain_text_body_with_email_inline_images
         return @plain_text_body unless @plain_text_body.nil?
-        part = email.text_part || email.html_part || email
+        part = email.html_part || email.text_part || email
         @plain_text_body = Redmine::CodesetUtil.to_utf8(part.body.decoded, part.charset)
     
         email_images = {}
-        email.part.each do |part|
+        email.all_parts.each do |part|
```

Comment:
In most cases (as far as I can say), inline images are included into the "related" part of the email, which is inside the "alternative" part. And, `email.part` returns only top-level parts (i.e., the "alternative" one). So, the plugin should check all the available parts, including nested ones.